src/main/java/org/stellar/sdk/xdr/Memo.java

```diff
@@ -15,7 +16,7 @@
 //  case MEMO_NONE:
 //      void;
 //  case MEMO_TEXT:
-//      string text<28>;
+//      opaque text<28>;
```

Comment:
At the top of this file is a comment saying the file file is generated with [xdrgen], but I don't see any updates to xdrgen that would have caused this. Are these changes being made manually to this file and the .x file in this repository without changing xdrgen? We won't be able to use xdrgen to generate these files again without losing these changes.
https://github.com/stellar/java-stellar-sdk/blob/9739b4bd877e4c72809d1f7da7e56a250c0cd4d1/src/main/java/org/stellar/sdk/xdr/Memo.java#L1-L2

[xdrgen]: https://github.com/stellar/xdrgen

I think we should keep this as `string` and change xdrgen so that it internally stores the field as a `byte[]` and offers up both `getText() : String` and `getTextBytes() : byte[]` functions, and similar setters, which should be backwards compatible and a non-breaking change to code it generates.

Reply:
see the discussion for this issue: https://github.com/stellar/go/issues/2022

Reply:
I DM'd with @tamirms and he confirmed this isn't a manual edit and the file generated by xdrgen, which was the main thing I was concerned about. I was misunderstanding the generated functions the byte type exports. Nice!